src/App.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react';
+import Courses from '../src/components/Courses/Courses';
+import Header from './components/Header/Header';
+import { mockCourses } from './data/data';
+import CreateCourse from './components/CreateCourse/CreateCourse';
+
+function App() {
+	const [course, setCourse] = React.useState();
```

Comment:
I would say this indentation is too big. Usually, devs are setting `tabWidth` equals `2` in prettier settings.